script.simkl/addon.xml

```diff
@@ -24,5 +24,9 @@
       <fanart>resources/fanart.jpg</fanart>
     </assets>
     <language>en</language>
+    <news>
```

Comment:
the news tag is only available for krypton and later versions

Reply:
Although I am ok with keeping it in the addon.xml since the tag will be ignored by jarvis

Reply:
Doesn't krypton and later versions cannot install the addon? so they will use this tag or am I wrong?

Reply:
This add-on can be installed in Krypton and newer versions. Those versions do not have a `changelog.txt` fallback, so if you want to see a changelog inside Kodi, you have to keep the news tag.

Reply:
yeap, that is what I wanted to know. We left it specifically for this. Thanks.